eclass/kernel-build.eclass

```diff
@@ -213,6 +206,32 @@ kernel-build_src_install() {
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/build
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/source
 
+	# Install the unstripped uncompressed vmlinux for use with systemtap
+	# etc.  Use mv rather than doins for the same reason as above --
+	# space and time.
```

Comment:
Space-time continuum? ;-)

Reply:
Oh, that should be removed since there is no `mv` any more.